src/languages/de-DE.json

```
@@ -0,0 +1,278 @@
+
+	"serverinfo_server_stats": "**Server stats**",
+	"serverinfo_channel_stats": "**Channel stats**",
+	"serverinfo_booster_stats": "**Booster stats**",
+	"serverinfo_server_stats_total": "Total: **{guild_members}**/{guild_max}\n",
+	"serverinfo_server_stats_online": "{emote_online}: **{guild_online}**\n",
+	"serverinfo_server_stats_idle": "{emote_idle}: **{guild_idle}**\n",
+	"serverinfo_server_stats_dnd": "{emote_dnd}: **{guild_dnd}**\n",
+	"serverinfo_server_stats_offline": "{emote_offline}: **{guild_offline}**\n",
+	"serverinfo_channel_stats_voice": "Voice Channels: **{guild_voice}**\n",
+	"serverinfo_channel_stats_text": "Text Channels: **{guild_text}**\n",
+	"serverinfo_channel_stats_category": "Category: **{guild_category}**\n",
+	"serverinfo_booster_tier": "Booster Level: **{guild_booster_tier}**\n",
+	"serverinfo_booster_boosters": "Booster: **{guild_booster_boosters}**\n",
+	"serverinfo_booster_tier_1": "Emoji slots: **100**\nAudio quality: **128** kbps\n",
+	"serverinfo_booster_tier_2": "Emoji slots: **150**\nAudio quality: **256** kbps\nUpload limit: **50 MB**\n",
+	"serverinfo_booster_tier_3": "Emoji slots: **250**\nAudio quality: **384** kbps\nUpload limit: **100 MB**\n",
+	"serverinfo_roles": "Rollen ({guild_amount_roles})",
+	"serverinfo_emotes": "Emojis ({guild_amount_emotes})",
+	"serverinfo_roles_too_many": " und **{guild_roles_left}** mehr",
+	"serverinfo_emotes_too_many": " und **{guild_emotes_left}** mehr",
+	"serverinfo_emotes_none": "Keine Emojis",
+
+	"ban_success": "{emote_success} **{target_tag}** `({target_id})` wurde für **{reason}** `[#{infractionId}] gebannt`",
+	"ban_fail": "{emote_warn} Nichtmöglich **{target_tag}** `({target_id})` zu bannen",
+	"already_banned": "{emote_warn} **{target_tag}** `({target_id})` Ist bereits mit folgender Begründung gebannt: `{reason}`",
+
+	"not_banned": "{emote_warn} **{target_tag}** `({target_id})` ist noch nicht gebannt",
+	"unban_success": "{emote_success} **{target_tag}** `({target_id})` wurde für **{reason}** `[#{infractionId}]` entbannt",
+
+	"kick_success": "{emote_success} **{target_tag}** `({target_id})` wurde für **{reason}** `[#{infractionId}]` gekickt",
+	"kick_fail": "{emote_warn} Nichtmöglich **{target_tag}** `({target_id})` zu kicken",
+
+	"purge_too_many": "{emote_warn} Ich kann nur 200 Nachrichten gleichzeitig löschen",
+	"purge_too_few": "{emote_warn} Du kannst nicht weniger als eine Nachricht von mir löschen lassen :confused: Warum sollte man sowas überhaupt machen",
+
+	"verification_non_integer": "{emote_warn} Level benötigt zum Integrieren",
+	"verification_community_zero": "{emote_warn} Verifikations Level für Community-Server nicht unter `0` gesetzt werden!",
+	"verification_level_error": "{emote_warn} Das Verifikations Level kann nicht höer als `4` oder niedriger als `0` gesetzt werden",
+	"verification_level_success": "{emote_success} Das Verifikations-Level wurde erfolreich zu  `{level}` geändert",
```

Discord Lingo, doublespace.
```suggestion
	"verification_level_success": "{emote_success} Das Verifizierungsstufe wurde erfolreich auf `{level}` gesetzt",
```